doc/api/next_api_changes/deprecations/17662-TAC.rst

```diff
@@ -2,8 +2,8 @@ Increase minimum supported versions of Python and dependencies
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 
-For Maptlotlib 3.4 the :ref:`minimum supported versions
-<install_requirements>` are being bumped
+For Maptlotlib 3.4 the :ref:`minimum supported versions <dependencies>`
```

Comment:
So does this get manually updated on each new build? (Alt: for the latest release)

Reply:
What do you mean by "this"?

This is an API change note, which is specific to the release and does not need update.

The list of dependencies is manually maintained under the link below. It's manually maintained as before. I've only moved it to it's own section instead of having it hidden in the install docs.

Reply:
Oh sorry I didn't look at the file name 😅 yeah it's fine then!